.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,29 @@
+name: ci-build
+on:
+  push:
+    branches:
+      - master
+    tags-ignore:
+      - '*.*'
+    pull_request:
+      branches:
+        - master
+
+jobs:
+  golangci:
+    name: Test
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Set XDG_RUNTIME_DIR
+        run: echo "XDG_RUNTIME_DIR=/run/user/$UID" >> $GITHUB_ENV
```

Comment:
out of curiosity, why do you need this step?

Reply:
Apparently this is one of the things required in order to expose the Podman socket.  I'm still trying to test this though.